src/ui_ng/src/app/project/create-project/create-project.component.ts

```diff
@@ -111,7 +111,7 @@ export class CreateProjectComponent implements AfterViewChecked, OnInit, OnDestr
   }
 
   onSubmit() {
-    this.btnIsOk=false;
+    this.isSubmitValid=false;
```

Comment:
.subscribe(
  ....

  response=> this.onGoing = false;

  error=> this.onGoing = false;
)